src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -621,9 +769,17 @@ public void setPort(int port) {
     public int getPort() {
         return port;
     }
+    
+    public void setHostAddresses(String hostAddresses) {
```

Comment:
I'm debating whether I like a comma separated list of addresses, or a first-class list better.

For example, you could have:
```
/*
 * Add the given "hostname:port" to the list of possible destination hosts to which to send logs.
 * Hosts will be tried in the order in which they are added.
 * Logs will be sent to the first one which accepts the connection.
 */
void addHostAddress(String hostAddress)
```

And then have your logback config be this:
```
<appender....>
    <hostAddress>hostname1:port1</hostAddress>
    <hostAddress>hostname2:port2</hostAddress>
</appender>
```

Having separate `hostAddress` elements looks better to me, as long as you're not building up a list via variables in xml.